src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java

```diff
@@ -271,6 +271,19 @@ static <T extends LfElement> void updateRemoteVoltageControlEquations(VoltageCon
                             .orElseThrow()
                             .setActive(false);
                 }
+
+                List<T> disabled = voltageControl.getMergedControllerElements()
```

Comment:
Because we re-create distribution equation without taking into account the status of the controller elements, we have to add this in a way.